File: dsa-Python/Pattern/pattern7.py
# This code prompts the user to enter a limit and stores it in the variable `limit`. It then
# initializes the variable `count` to 1.
limit=int(input("Enter the limit "))
count=1
for i in range(1,limit+1,1):
    print("  "*(limit-i)+"* "*count)
    count+=2

File: dsa-Python/Pattern/pattern7.java
/**
 * The pattern7 class prints a pattern of asterisks in a pyramid shape based on the user input for the
 * number of rows.
 */
import java.util.*;
public class pattern7 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println( "Enter the number of rows:");
        int limit=sc.nextInt();
        int count=1;
        for (int i = 1; i <=limit ; i++) {
            for(int j=1;j<=limit-i;j++){
                System.out.print("  ");
            }
            for(int k=1;k<=count;k++){
                System.out.print("* ");
            }
            System.out.println();
            count+=2;
        }
        sc.close();
    }
}
